kamrons-game.py

```diff
@@ -42,33 +42,99 @@ def get_answer(prompt):
+                                        if answer == "home":
+                                            print("It looks like this crazy adventure is coming to an end. It is now evening at your house and you look out the window, stare into space, and pass out before your head hits the pillow.")
+                                            print("THE END")
+                                        
+                                        if answer == "join":
+                                            print("You decide to join them on their investigation. You offer them your address to investigate for the stolen money. They let you into their vehicle when you realize that... they aren't going to the right place! You tell them the address again, the driver nods, but doesn't change directions. Then it hits you: These people aren't the real FBI! You panic in your brain but stay completely still. At any moment they could kill you and no one would ever know! You manage to calm yourself and think that maybe if you excuse yourself, you can call the cops and they will handle it! So you do just that.")
+                                            print("You excuse yourself to the bathroom at a resturant and manage to call the cops. The cops come and recognize that the FBI agents don't have proper badges. The fake FBI agents are now in prison! And thanks to you. The cops reward you with... a ride home. An actual one. So you live happily ever after and this adventure became only a story that you tell at parties.")
+                                            print("We regret to inform you that this story has come to a close, but if you would like to play again and discover all the endings and funny passages, go ahead and type 'yes' down below. Now without further ado, I'm just going to say, THE END, and thank you for playing.")
+                                        
+                                    else:
+                                        print("Invalid command. You stood there like a dummy until nightfall when the FBI left, and then you just stood there for the rest of your life, slowly rotting to a crisp of ash.") 
+                                        
+                                else:
+                                     print("Invalid command. You spoke nothing or did nothing and the FBI just assumed you guilty and killed you, case closed.")
+                                        
+                            else:
+                                print("Invalid command. You did nothing for the rest of your life and died of either starvation of old age. I wouldn't know.")
+                                        
                         else:
-                            print("You did nothing and the FBI ran over you. Game over.")
+                            print("Invalid command. You did nothing and the FBI ran over you. Game over.")
```

Comment:
Instead of `Invalid command`, maybe something like `It seems like you are confused because that wasn't an option.`

Reply:
on it, good idea